geopandas/tools/geocoding.py

```diff
@@ -169,16 +169,22 @@ def _prepare_geocode_result(results):
     index = []
 
     for i, s in results.items():
-        address, loc = s
 
-        # loc is lat, lon and we want lon, lat
-        if loc is None:
+        if s is None:
             p = Point()
+            address = np.nan
+
         else:
-            p = Point(loc[1], loc[0])
+            address, loc = s
 
-        if address is None:
-            address = np.nan
+            # loc is lat, lon and we want lon, lat
+            if loc is None:
+                p = Point()
+            else:
+                p = Point(loc[1], loc[0])
+
+            if address is None:
+                address = np.nan
```

Comment:
```suggestion
```

I would leave it at `None` -> pandas also sees that as missing value (just as np.nan), and that's what we did in the past